base/uk.ac.stfc.isis.ibex.instrument/src/uk/ac/stfc/isis/ibex/instrument/internal/PVPrefix.java

```diff
@@ -19,38 +19,118 @@
 
 package uk.ac.stfc.isis.ibex.instrument.internal;
 
+import com.google.common.collect.ImmutableList;
+
+/**
+ * The Class PVPrefix representing the prefix for the instrument.
+ */
 public class PVPrefix {
```

Comment:
PVPrefix is a combination of:

- A string wrapper
- A factory for generating PV prefixes

Having discussed with John, this would be better as:

- Not having a PV prefix class at all. It's only ever converted back to a string as soon as it's created. There's currently no value in the class
- Putting the remaining logic as a PVPrefixFactory class that handles the logic of PV prefix generation. An instance of the factory can then be injected in the InstrumentInfo class if we ever want to test it in isolation